libsrc/CMakeLists.txt

```diff
@@ -49,6 +51,11 @@ add_library(${lib_name} STATIC
   ${ADDITIONAL_SRCS}
   )
 
+CHECK_CXX_COMPILER_FLAG("-fPIC" CXX_HAS_FPIC)
+if(CXX_HAS_FPIC)
+  target_compile_options(${lib_name} PRIVATE "-fPIC")
+endif()
+
```

Comment:
Instead, consider using this variable: https://cmake.org/cmake/help/v3.5/variable/CMAKE_POSITION_INDEPENDENT_CODE.html